admin-dev/themes/new-theme/scss/components/layout/_content_div.scss

```diff
@@ -38,6 +38,14 @@
   }
 }
 
+// Special case for customer page the table has too many columns to fit the screen so we allow an horizontal scroll
```

Comment:
Shouldn't it go in https://github.com/PrestaShop/PrestaShop/tree/develop/admin-dev/themes/new-theme/scss/pages ?